mage/helm/lib.go

```diff
@@ -12,6 +12,9 @@ import (
 func Install(chartPath string, releaseName string, namespace string, overrides map[string]string) error {
 	args := []string{
 		"install",
+		// k8s 1.15 does not support all the validations we have setup on 
+		// the CRD, so we just turn off validation. What could go wrong?
+		"--disable-openapi-validation",
```

Comment:
If only the old versions of things died and were gone forever.

Reply:
😬 I don't feel good about this - what happens if you remove it?

Reply:
It will fail to install in 1.15.

Reply:
ok - just add a FIXME, I'm working on better handling multiple k8s versions in our helm chart right now, so I'll come back here when that's ready